console/src/main/scripts/plugins/metrics/html/app-details/detail-platform.html

```diff
@@ -1,187 +1,204 @@
+                </hawkular-chart>
+              </span>
+              <span class="hk-data" ng-show="!os.chartCpuDataMulti[cpu]">Loading ...</span>
+              <span class="hk-item">{{os.processorListNames[cpu]}}</span>
+            </div>
           </div>
         </div>
+      </div>
 
-        <div class="panel panel-default hk.graph" ng-if="os.resolvedChartFileSystemData[fs.id]">
-          <div class="hk-graph-container hk-graph-metrics">
-            <!-- HINT: colors for the chart can be changed in the hawkular-charts.css -->
-            <hawkular-chart
-                  multi-data="os.chartFileSystemData[fs.id]"
-                  chart-type="multiline"
-                  y-axis-units="FS Usage (MB)"
-                  chart-height="250">
-            </hawkular-chart>
-          </div>
-          <div class="hk-legend text-left hk-legend-metrics">
-            <div class="col-md-12">
-              <h4>Connection Availability</h4>
-              <div class="hk-graphselector hk-orange">
-                <input id="connavail_{{$index}}" type="checkbox"
-                     ng-checked="!os.skipChartData[fs.id + '_Total']"
-                     ng-click="os.toggleChartData(fs.id + '_Total')">
-                <label for="connavail_{{$index}}"><span>Total space</span></label>
-              </div>
-              <div class="hk-graphselector hk-blue">
-                <input id="conninuse_{{$index}}" type="checkbox"
-                     ng-checked="!os.skipChartData[fs.id + '_Free']"
-                     ng-click="os.toggleChartData(fs.id + '_Free')">
-                <label for="conninuse_{{$index}}"><span>Free space</span></label>
+      <div class="panel panel-default">
+        <div class="hk-graph">
+          <h2>File System usage</h2>
+        </div>
+      </div>
+        <div class="hk-resource-container"
+             ng-repeat="fs in os.fileStoreList | filter: os.filter| orderBy:'name':osSortInv">
+
+          <div class="panel panel-default hk-summary">
+            <div class="panel-heading">
+              <span class="hk-heading hk-bold hk-resource-name">
+                {{fs.name}}
+              </span>
+              <div class="btn-group pull-right hk-action-buttons" dropdown auto-close="always">
+                <button id="ds-actions" class="btn btn-link" type="button" dropdown-toggle aria-haspopup="true"
+                        aria-expanded="true">
+                  Actions
+                  <span class="caret"></span>
+                </button>
+                <ul class="dropdown-menu" role="menu" aria-labelledby="ds-actions">
+                  <li class="disabled"><a href="#">Enable</a></li>
+                  <li class="disabled"><a href="#">Disable</a></li>
+                </ul>
               </div>
             </div>
+            <div class="row"></div>
+          </div>
+          <div class="hk-panel-collapse-area" ng-class="{'hk-collapsed' : !os.expandedList[fs.id]}">
```

Comment:
No need to change for now, but let's try and avoid expressions such as `os.expandedList[fs.id]` because they avoid type safety. expandedList should have an interface that provides proper code completion in the IDE and also would pick up any changes to it in the future and report at compile time (instead of runtime). Angular 2.0 will actually give us type safety in the view.